src/openfluid/ware/SimulatorSignature.cpp

```diff
@@ -70,5 +72,25 @@ SignatureDataItem::SignatureDataItem(const std::string& N, const std::string& D,
   }
 }
 
+
+// =====================================================================
+// =====================================================================
+
+
+std::vector<std::string> SimulatorSignature::getTagsByType(const std::string& Type) const
+{
+  std::vector<std::string> MatchingTags;
+  for(const auto& Tag : Tags)
+  {
+    if(openfluid::tools::contains(Tag, Type, false) && openfluid::tools::startsWith(Tag, Type))
```

Comment:
Why is the first part still useful? 
And we would prefer to add the separator in the check to ensure we filter "domain::foo" and not "domain_of_foo"